scripts/verify-literal-check.py

```diff
@@ -0,0 +1,100 @@
+import os
+import argparse
+import re
+
+import urllib.request
+
+# # URL_REGEX FROM:
+# https://mathiasbynens.be/demo/url-regex
+
+
+URL_REGEX = re.compile("((https?):((//)|(\\\\))+([\w\d:#@%/;$()~_?\+-=\\\.&](#!)?)*)", re.DOTALL)
+
+# Grab RemoteLiteralIncludes from .rst resources
+# Return the URLs in the RLIs
+def grab_rli_urls(path: str):
+
+    # retrieve list of rst documents
+    documents = {}
+    for root, subdirs, files in os.walk(os.path.abspath(path)):
+        for document in files:
+            if ".rst" in document:
+                documents[os.path.join(root, document)] = os.path.join(root, document)
```

Comment:
They key and value are the same so it can just be a list/set/iterable?
So, something like this would be simpler.
```suggestion
from pathlib import Path # before
    documents = Path(path).glob("**/*.rst")
```